### PR TITLE
kids fix

### PR DESCRIPTION
**Pull request by Maquiner**
fixed by invoking self. inside method

### CODE REVIEW

**Review by damonge — CHANGES_REQUESTED** [review]
Were we gonna save the masks too?

File: xcell/mappers/mapper_KV450.py [hunk 1]
```diff
@@ -200,24 +202,40 @@ def get_signal_map(self, mode=None):
...
             self.masks[kind] = get_map_from_points(data, self.nside,
                                                    w=data['weight'],
                                                    ra_name='ALPHA_J2000',
                                                    dec_name='DELTA_J2000')
+            hp.write_map(fname_lite, self.masks[kind], overwrite=True)
         self.mask = self.masks[kind]
         return self.mask
 
+    def _get_w2s2(self, mode):
+        kind, e1f, e2f, mod = self._set_mode(mode)
```
Comment (damonge) [inline xcell/mappers/mapper_KV450.py:226]: oh, can you do the same thing with this map?

**Review by damonge — CHANGES_REQUESTED** [review]

File: xcell/mappers/mapper_KV450.py (hunk 1 above)
Comment (Maquiner) [reply xcell/mappers/mapper_KV450.py:227]: done!

**Review by damonge — APPROVED** [review]
LGTM!

**Pull request merged**